src/main/resources/swagger/api-docs.yaml

```diff
@@ -135,9 +135,7 @@ paths:
       summary: gets a key for the user's pet service account, get_pet_private_key action on cloud-extension/google required
       responses:
         200:
-          description: 'key for a pet service account'
-          schema:
-            $ref: '#/definitions/ServiceAccountKey'
+          description: 'json format key for a pet service account, see https://cloud.google.com/iam/docs/creating-managing-service-account-keys'
```

Comment:
Should this have 

> schema:
>              type: string

Reply:
I don't think so... it is actual json but I don't want to specify what is in it because it is google specific.

Reply:
you're right!